spec/support/i18n_helper.rb

```diff
@@ -0,0 +1,13 @@
+module I18n
+  class << self
+    prepend(
+      Module.new do
+        def t(*args, ignore_test_helper_missing_interpolation: false, **kwargs)
```

Comment:
👏 
Can you add a test for this specific behavior (raising if a translation has a missing interpolation)? This seems significant enough that we want to make sure it doesn't regress.

Reply:
> Can you add a test for this specific behavior (raising if a translation has a missing interpolation)? This seems significant enough that we want to make sure it doesn't regress.

I'm open to it! Not exactly sure where a spec for a spec helper should live, though, folder-wise 😄

Reply:
AH I completely missed that this was in a helper file. All good!

Reply:
Oh yeah, this is just for specs. It could be an option to consider having this in the runtime for much higher visibility to the problem, but I'm not as confident we'd want that to risk that much disruption to user flows.